usd/src/usd_parser/USD2SDF.hh

```diff
@@ -15,8 +15,8 @@
  *
 */
 
-#ifndef USD_PARSER_USD2SDF_HH_
-#define USD_PARSER_USD2SDF_HH_
+#ifndef SDF_USD_USD_PARSER_USD2SDF_HH_
+#define SDF_USD_USD_PARSER_USD2SDF_HH_
```

Comment:
Since this is a source file, I actually think that the original include guard (`USD_PARSER_USD2SDF_HH_`) is okay - according to https://github.com/ignitionrobotics/sdformat/pull/817#discussion_r791031195, the include guard should be the path of the file after either include or src.